### DISCUSSION TITLE
Lost frames

### DISCUSSION THREAD

**Discussion by DiagnozHub** [Motion-Project/motion#1599] [General]
Hello team,

Now i'm trying to reseach Motion project.
I found out one thing:
0. My camera fps = 22, my stream is rtsp.
1. I set fps = 10.
2. I set record video files in mp4 format.
3. I downloaded result file.

And I saw that we lost a lot of frames. In my opinion we lost a lot of frames. You can se it in attached file.

https://user-images.githubusercontent.com/117123970/202876295-dd979ac1-6d7d-417b-8fa7-2284a2fc5081.mp4

But when i see live video the fps and frames are good.

**Comment by Mr-Dave** [comment]
If your camera is at 22fps, why would you not set the framerate and capture_rate at 22?  Is the computer capable of handling the processing?

**Comment by DiagnozHub** [comment]
I try reduce load.

And if we have possibility reduce fps in settings we could use this option, is not?
